frontend/components/common/other/AnimatedArrow.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+    <UpArrow :style="style" />
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import UpArrow from '~/assets/icons/uparrow.svg';
+
+export default Vue.extend({
+    components: {
+        UpArrow,
+    },
+    props: {
+        size: {
```

Comment:
I would maybe make the `size` prop a string and allow the dev to define what level of measurement to use. This would allow us to use px, rem or any other kind of unit. 

This change will need to be taken into account in the computed style data as well!